app/src/main/res/values/styles.xml

```diff
@@ -247,7 +247,7 @@
 
   <style name="Heading4" parent="TextView.Common1">
     <item name="android:fontFamily">sans-serif</item>
-    <item name="android:textSize">18sp</item>
+    <item name="android:textSize">16sp</item>
```

Comment:
@ArchitJain1201 also I think this heading 4 value should not be altered. Can you pls explain the reason for this change?